prefetch.bs

```diff
@@ -539,6 +559,9 @@ Modify the [=snapshot source snapshot params=] algorithm to set the return value
             1. If <var ignore>documentResource</var> is null:
               1. Let |prefetchRecord| be the result of [=finding a matching complete prefetch record=] given <var ignore>navigable</var>, <var ignore>sourceSnapshotParams</var>, <var ignore>entry</var>'s [=session history entry/URL=], and <var ignore>sourceSnapshotParams</var>'s [=source snapshot params/sandboxing flags=].
               1. If |prefetchRecord| is not null:
+                1. Set |request|'s [=request/reserved client=] to the result of [=creating a reserved client=] given <var ignore>navigable</var>, |request|'s [=request/current URL=], and null.
```

Comment:
(Probably not specific to this PR)
`navigable` here when prefetching is the navigable of the initiator Document, which is not necessarily the same navigable used when actual navigation using the prefetched result (=target navigable, which can be different from the initiator Document when `window.open` etc.). Is this OK?